playground/index.html

```diff
@@ -14,11 +14,16 @@
   <script src="https://unpkg.com/whatwg-fetch@3.0.0/dist/fetch.umd.js"></script>
   <script src="https://unpkg.com/popper.js@1.16.0/dist/umd/popper.min.js"></script>
   <script src="https://unpkg.com/bootstrap@4.4.0/dist/js/bootstrap.min.js"></script>
+  <script src="https://unpkg.com/showdown/dist/showdown.min.js"></script>
```

Comment:
I'd link to a specific version (https://unpkg.com/showdown@1.9.1/dist/showdown.min.js) otherwise this might break in future!